cache/nvm_cache.h

```diff
@@ -0,0 +1,299 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+//
+// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+#pragma once
+
+#include <string>
+
+#include "cache/sharded_cache.h"
+
+#include "port/malloc.h"
+#include "port/port.h"
+#include "util/autovector.h"
+
+#include <unistd.h>
+#include <libpmemobj++/p.hpp>
+#include <libpmemobj++/container/concurrent_hash_map.hpp>
+#include <libpmemobj++/pool.hpp>
+#include <libpmemobj++/persistent_ptr.hpp>
+
+using namespace pmem::obj;
+
+namespace ROCKSDB_NAMESPACE {
+
+// This will be the transient handle of the cache.
+struct TransientHandle {
```

Comment:
This is the same as `LRUHandle`?

Reply:
This will not be shortly. Best bet, it can be derived from `LRHandle` but I'm not sure at this moment.